tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AutoInstrumentationExtensions.cs

```diff
@@ -28,5 +32,25 @@ internal static void DisposeWithException(this Scope scope, Exception exception)
                 }
             }
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void ServerlessDispose(this Scope scope)
```

Comment:
Instead of adding a global `Dispose()` variant that is specifically for Serverless, I would prefer if we move this method into one of the Serverless classes and not make it an extension method (i.e. remove `this` in the parameter list).

The `DisposeWithException()` above is actually shared between all instrumentations.

Reply:
✅